registration/views.py

```diff
@@ -549,6 +561,20 @@ def get_context_data(self, form, **kwargs):
         d['postal_code'] = self.request.session[SESSION_POSTAL_CODE]
         d['email'] = self.request.session[SESSION_EMAIL]
 
+        form_test = self.form_list[unicode(self.steps.current)]
+        if 'license_id' in form_test.base_fields:
+            if SESSION_LICENSE_ID_FORMATS in self.request.session:
+                license_id_formats = [str(license_id_format) for license_id_format in
```

Comment:
@mabuaisha you should pull out the format conversion into a separate method that can be properly unit tested on its own.